site/pages/create-consequence-operator/[disruptionId]/[consequenceIndex].page.tsx

```diff
@@ -209,7 +209,7 @@ const CreateConsequenceOperator = (props: CreateConsequenceOperatorProps): React
                                     props.disruptionSummary ? stateUpdater(props.disruptionSummary, "description") : "";
```

Comment:
Can we change the name of this prop to disruptionDescription to reflect it is copy the description not a summary